.circleci/config.yml

```diff
@@ -16,13 +16,17 @@ jobs:
       - checkout
       - restore-cache:
           keys:
-            - 'dependencies-{{ .Branch }}-{{ checksum "package.json" }}'
-            - 'dependencies-{{ .Branch }}-'
-      - run: yarn --no-progress
+            # when lock file changes, use increasingly general patterns to restore cache
+            # https://circleci.com/docs/2.0/caching/
+            - yarn-packages-v1-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - yarn-packages-v1-{{ .Branch }}-
+            - yarn-packages-v1-
+      - run: 'sudo yarn global add greenkeeper-lockfile@next'
```

Comment:
Is sudo needed? I've never had a need for it when running global yarn installs before. Is it something about the package?

Reply:
@camacho This was from the greenkeeper doc. Let me try without.

Reply:
OK, this should be much cleaner now.